README.md

```diff
@@ -4,6 +4,9 @@ A GitHub Action which installs and configures the LXD snap on a runner.
 
 By default, this action will install LXD from the `latest/stable` snap channel. You can specify a different channel using the `channel` input. (See `snap info lxd` for a list of available channels).
 
+The action will also refresh LXD if it is already installed on a runner from SNAP,
+it can be skipped using the option 'refresh: false'.
```

Comment:
Some nits:
- "refresh the LXD snap" might be clearer
- "SNAP" -> "snap"
- use backticks \` \` so code blocks display correctly.

Reply:
Done.